Lib/asyncio/coroutines.py

```diff
@@ -218,23 +156,20 @@ def coro(*args, **kw):
             if (base_futures.isfuture(res) or inspect.isgenerator(res) or
                 isinstance(res, CoroWrapper)):
                 res = yield from res
-            elif _AwaitableABC is not None:
+            elif Awaitable is not None:
```

Comment:
Is `Awaitable` imported?

Reply:
Good catch. Fixed.

Reply:
Isn't `Awaitable is not None` always true?